src/benchmarklib/benchmark_runner.cpp

```diff
@@ -518,6 +518,7 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
   // method and not a class, retrieving this default value properly would require some major refactoring of how
   // benchmarks interact with the BenchmarkRunner. At this moment, that does not seem to be worth the effort.
   const auto* const default_mode = (benchmark_name == "TPC-C Benchmark" ? "Shuffled" : "Ordered");
+  const auto* const default_table_indexes = (benchmark_name == "TPC-C Benchmark" ? "true" : "false");
```

Comment:
Should we also document this somewhere else (not sure where tbh)?

Reply:
Hmm, when running the benchmark it should be rather clear. I'll add a short comment the line above?